examples/cae/cae-slurm.yaml

```diff
@@ -208,7 +208,7 @@ deployment_groups:
       node_count_dynamic_max: 10
       machine_type: c3-highmem-176
       disk_type: 'pd-balanced'
-      bandwidth_tier: 'tier_1_enabled'
+      bandwidth_tier: gvnic_enabled
```

Comment:
Having gvnic enabled is a pre-req to having tier 1 enabled. In addition you can think of tier 1 as an extra performance beyond gvnic. All that to say, places where we see tier 1 enabled, I would assume someone has already put some thought into it and I would leave it as tier 1.

Reply:
Makes sense. I changed the ones originally marked as tier 1 back.